proto/akash/deployment/v1beta1/query.proto

```diff
@@ -1,55 +1,62 @@
 syntax = "proto3";
-package akash.deployment;
+package akash.deployment.v1beta1;
 
 import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
-import "akash/deployment/deployment.proto";
-import "akash/deployment/group.proto";
+import "akash/deployment/v1beta1/deployment.proto";
+import "akash/deployment/v1beta1/group.proto";
 
 option go_package = "github.com/ovrclk/akash/x/deployment/types";
 
 // Query defines the gRPC querier service
 service Query {
     // Deployments queries deployments
-    rpc Deployments (QueryDeploymentsRequest) returns (QueryDeploymentsResponse) {}
+    rpc Deployments (QueryDeploymentsRequest) returns (QueryDeploymentsResponse) {
+        option (google.api.http).get = "/akash/deployment/v1beta1/deployments/list";
+    }
 
     // Deployment queries deployment details
-    rpc Deployment (QueryDeploymentRequest) returns (QueryDeploymentResponse) {}
+    rpc Deployment (QueryDeploymentRequest) returns (QueryDeploymentResponse) {
+        option (google.api.http).get = "/akash/deployment/v1beta1/deployments/info";
+    }
 
     // Group queries group details
-    rpc Group (QueryGroupRequest) returns (QueryGroupResponse) {}
+    rpc Group (QueryGroupRequest) returns (QueryGroupResponse) {
+        option (google.api.http).get = "/akash/deployment/v1beta1/groups/info";
+    }
 }
 
 // QueryDeploymentsRequest is request type for the Query/Deployments RPC method
 message QueryDeploymentsRequest{
-    DeploymentFilters Filters = 1 [(gogoproto.nullable) = false];
+    DeploymentFilters filters = 1 [(gogoproto.nullable) = false];
 
     cosmos.base.query.v1beta1.PageRequest pagination = 2;
 }
 
 // QueryDeploymentsResponse is response type for the Query/Deployments RPC method
 message QueryDeploymentsResponse{
-    repeated DeploymentResponse Deployments = 1 [(gogoproto.nullable) = false,(gogoproto.castrepeated)="DeploymentResponses"];
+    repeated DeploymentResponse deployments = 1 [(gogoproto.nullable) = false,(gogoproto.castrepeated)="DeploymentResponses"];
```

Comment:
```suggestion
    repeated DeploymentResponse deployments = 1 [(gogoproto.nullable) = false, (gogoproto.castrepeated)="DeploymentResponses"];
```

Reply:
Let's add a formatting tool for proto files. `clang-format` is the best out there, that can be separate PR